oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoFullGcNodeBin.java

```diff
@@ -44,14 +44,19 @@
  *  Each method delegates directly to DocumentStore
  */
 public class MongoFullGcNodeBin implements FullGcNodeBin {
-    private static final Logger LOG = getLogger(MongoFullGcNodeBin.class);
     public static final String GC_COLLECTED_AT = "_gcCollectedAt";
+    private static final Logger LOG = getLogger(MongoFullGcNodeBin.class);
```

Comment:
Use `LoggerFactory.getLogger(MongoFullGcNodeBin.class)` instead of `getLogger(MongoFullGcNodeBin.class)`